clients/js/test/das.test.ts

```diff
@@ -0,0 +1,210 @@
+import test from 'ava';
+import { publicKey } from '@metaplex-foundation/umi';
+import {
+  isAssets,
+  searchAssets,
+  fetchAssetsByAuthority,
+  fetchAssetsByCollection,
+  fetchAssetsByOwner,
+  fetchCollectionByUpdateAuthority,
+} from '../src/das';
+import { createUmiWithDas, DAS_API_ENDPOINT } from './_setup';
+
+const TEST_MPL_CORE_ASSET = {
```

Comment:
If you have more ideas for tests, I would gladly implement them.

The problem with those tests is that they're not stable at all. E.g. if more collections are created with some update authority, the collections order will be changed, and taking a random collection from the result won't work, because I'll expect other asset to compare with hardcoded collection/asset data. Ideally, I should use `assertAsset`/`assertCollection`, which fetch assets via mpl core native functions, but `Too Many Requests` is guaranteed there and the tests will fail as well.

Reply:
@nhanphan @danenbm the tests were the biggest change.

1. Used my file wallet instead of Phantom
2. Left a function which I used for creation of assets and collections. So, I guess we will be able to recreate the same structure for assets/collections if needed
3. Added a couple of tests with assets/collections with different owner and update authority
4. Added some changes according to @danenbm's recent changes